test/features/caseView.feature

```diff
@@ -3,8 +3,6 @@ Feature: Set of scenarios testing case view functionality
 
   Background:
     Given I have logged in
-    And a case containing Tabs functionality exists
-    And I create the case
 
   Scenario: case reference visible on case view
     Then the case reference will be visible and formatted well
```

Comment:
don't forget to add the 2 deleted lines from the backround here - otherwise I will get confused when I come round to fixing this test :)

Reply:
Done.